src/sql/workbench/api/node/mainThreadNotebookDocumentsAndEditors.ts

```diff
@@ -361,26 +361,9 @@ export class MainThreadNotebookDocumentsAndEditors extends Disposable implements
 			pinned: !options.preview
 		};
 		let trusted = uri.scheme === Schemas.untitled;
-		let model = new NotebookInputModel(uri, undefined, trusted, undefined, undefined, undefined, options.connectionId);
-		let providerId = options.providerId;
-		let providers: string[] = undefined;
-		// Ensure there is always a sensible provider ID for this file type
-		providers = getProvidersForFileName(uri.fsPath, this._notebookService);
-		// Try to use a non-builtin provider first
-		if (providers) {
-			providerId = providers.find(p => p !== DEFAULT_NOTEBOOK_PROVIDER);
-			if (!providerId) {
-				providerId = model.providerId;
-			}
-		}
-		model.providers = providers;
-		model.providerId = providerId;
-		model.defaultKernel = options && options.defaultKernel;
```

Comment:
OK... so as mentioned above let's ensure all data previously set on the model is still set, by passing through via the NotebookInput